ffi/pic.lua

```diff
@@ -151,11 +151,12 @@ function GifDocument:openPage(number)
     return page
 end
 function GifDocument:close()
-    if giflib.DGifCloseFile(self.giffile, err) ~= giflib.GIF_OK then
+    if giflib.DGifCloseFile(self.giffile, self.err) ~= giflib.GIF_OK then
         error(string.format("error closing/deallocating GIF: %s",
```

Comment:
Strange it works with a non ffi typed variable.
I would use there, like it is used in the next function:
`local err = ffi.new("int[1]")  -- luacheck: no unused`